src/main.js

```diff
@@ -189,6 +189,11 @@ app.on('ready', function() {
         title: arg.title,
         content: arg.options.body
       });
+
+      /* Todo: add idle here */
```

Comment:
Exactly, and this is probably why this is the wrong place ;)
As we might at least want it for the other windows distributions.
Look at https://github.com/mattermost/desktop/blob/108cf91bf6023b7c13c58f6434ba36025debed9e/src/browser/webview/mattermost.js#L110 which should make it clear.